src/PageContent/Calendar/ContactContent.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import { AccountType, StringFormat } from '../../Enums';
+import './Content.css';
+import phone from '../../icons/phone.svg';
+
+class DescriptionContent extends Component {
+    constructor(props) {
+        super(props);
+        if (
+            this.props.delivery.donatingAgency &&
+            this.props.accountType === AccountType.DONATING_AGENCY_MEMBER
+        ) {
+            this.state = {
+                edit: false,
+                name: this.props.delivery.donatingAgency.contact.name,
+                phone: this.props.delivery.donatingAgency.contact.phone,
+                email: this.props.delivery.donatingAgency.contact.email
+            };
+        } else if (
+            this.props.delivery.receivingAgency &&
+            this.props.accountType === AccountType.RECEIVING_AGENCY
+        ) {
+            this.state = {
+                edit: false,
+                name: this.props.delivery.receivingAgency.contact.name,
+                phone: this.props.delivery.receivingAgency.contact.phone,
+                email: this.props.delivery.receivingAgency.contact.email
+            };
+        } else {
+            this.state = {
+                edit: false
+            };
+        }
```

Comment:
there will always be `donatingAgency` and `receivingAgency`, instead of repeating and expanding data out, i'd suggest something like:
```
this.state = {
    edit: false,
    contact: {},
    membersList: null
};
if (this.props.accountType === AccountType.DONATING_AGENCY_MEMBER) {
    this.state.contact = this.props.delivery.donatingAgency.contact;
} else if (this.props.accountType === AccountType.RECEIVING_AGENCY) {
    this.state.contact = this.props.delivery.receivingAgency.contact;
}
```